### DISCUSSION TITLE
Possible to edit domain?

### DISCUSSION THREAD

**Discussion by hydrandt** [plausible/analytics#1251] [Anything Else]
Hi, we are using hosted plausible (on plausible.io). I moved a website from new.ark.greensteps.cn to ark.greensteps.cn. It would be great to be able to rename the domain, so that we don't lose historical data. The old domain is now 302ing to the new domain. 

Is something like rename possible? Or do I have to nuke it and start collecting from scratch?

**Comment by Vigasaurus** [comment]
The domain listed is entirely cosmetic and as long as the `data-domain` matches, it's fine, so you don't *have to* change it if you don't want to. I'd imagine if you do want to change it that could be arranged, but don't quote me on that.

**Comment by ukutaht** [comment]
Hey @hydrandt!

Migrating the data over from one domain to the other is a manual job at the moment. We're happy to do it for customers of Plausible Cloud (plausible.io). Instructions on how to change the domain name: https://plausible.io/docs/change-domain-name
